templates/block-php.mustache

```diff
@@ -0,0 +1,13 @@
+<?php
+
+function {{machine_name}}_enqueue_block_editor_assets() {
+	wp_enqueue_script( '{{slug}}-block', plugins_url( '{{slug}}/block.js', __FILE__ ), array(
+		'wp-blocks',
+		'wp-i18n',
+		'wp-element',
+	), time() );
```

Comment:
It's probably a bad idea to use `time()` as the script version in production. If you want this to automatically update when the file is changed, you could use `filemtime()` to get the modified time for the file and append this instead.

Reply:
I copied it from one of the examples. Agreed on `filemtime` :)